charts/opensearch/CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [1.4.3]
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+- Fixed links to values.yaml in README.md.
```

Comment:
Looks like it got missed. We need to add the CHANGELOG

Reply:
Im actually not 100% sure how these changes got pulled in. They appeared when I added my signature to the commit.

Reply:
@bluehawk27 May be worthwhile bumping the chart again, as this is likely a merge in from main. 

But, we should also have a corresponding CHANGELOG section for your changes (per @TheAlgo statement).